Preprocessing of metadata (#16)
MrLongbottom/P10-Article: sections/Dataset.tex

They store their news articles in a non-public database, where each article contains multiple metadata fields which describe some aspect of the data e.g. the author.
The dataset we use ranges from 2017 to 2019 and contains 139261 articles that use a vocabulary of 69192 unique words after preprocessing.

The metadata fields that we will be working with do have some problems that can be mitigated to a degree by preprocessing.
These problems are all related to some metadata values having only a few documents.
Since the metadata values will be used to group documents together and find common topics and words within grouped documents, metadata values that group too few documents are not very relevant and will therefore be removed.

In the following section, we describe each of the metadata fields which are analyzed.

\subsection{Author}
This field is for the author, who has written the article.
Each article only has a single author, so we do not account for multiple authors.
This field is fully observed within the dataset, meaning that every article has an author.

> There were originally $227$ different authors within the dataset.
> After filtering away authors that have written less that $14$ documents ($\tilde0.001\%$ of the total document set), 184 remain.

Reviewer: ```suggestion
Originally there were $227$ different authors within the dataset.
After filtering away authors that have written less than $14$ documents ($\tilde0.001\%$ of the total document set), 184 remain.
```

%and they are almost evenly distributed in the number of articles they have written.

\subsection{Category}
% what it is?
The category field describes a variety of different aspects.
A proportion of the categories contains which specific newspaper they belong to, e.g. 'Aalborg-Newspaper'.
Another proportion of the category field describes the overall subject of the document, such as 'Culture' and 'Sports-newspaper'.

> However there are also nonsensical categories such as '53. Frederik', that do not seem to describe the subject of the document.

Reviewer: ```suggestion
However, there are also nonsensical categories such as '53. Frederik', that does not seem to describe the subject of the document.
```

	\item PLACES/Danmark/Nordjylland/Aalborg/Lillevorde
	\item TOPICS/Religion/Christianity
\end{itemize}
About $80\%$ of the observed fields contain the 'PLACES' variable and $20\%$ use the 'TOPICS' variable.

> There are also a few other top-level taxonomies however they are not as informative and very rarely used.

Reviewer 2: ```suggestion
There are also a few other top-level taxonomies; however, they are not as informative and very rarely used.
```

> There are also a few other top-level taxonomies, however they are not as informative and very rarely used.

Reviewer: ```suggestion
There are also a few other top-level taxonomies however they are not as informative and very rarely used.
```

> Originally there were $1135$ different taxonomies however after removing taxonomies used by less than $4$ documents ($\tilde 0.01\%$ of the number of documents with taxonomies), $613$ remain.

Reviewer 2: Er det rent faktisk 0.01, og ikke 0.001 af taxonomy dokument sættet der giver 4? Ved hurtig hovedregning synes jeg 0.001 giver mere mening.

> Originally there were $1135$ different taxonomies however after removing taxonomies used by less than $4$ documents ($\tilde 0.01\%$ of the number of documents with taxonomies), $613$ remain.

Author: ```suggestion
Originally there were $1135$ different taxonomies however after removing taxonomies used by less than $4$ documents ($\tilde 0.001\%$ of the number of documents with taxonomies), $613$ remain.
```

> Originally there were $1135$ different taxonomies however after removing taxonomies used by less than $4$ documents ($\tilde 0.001\%$ of the number of documents with taxonomies), $613$ remain.

Reviewer 2: ```suggestion
Originally there were $1135$ different taxonomies; however, after removing taxonomies used by less than $14$ documents ($\tilde 0.001\%$ of the total document set), $345$ remain.
```
Kan ikke huske det specifikke nye antal dokumenter.

> Originally there were $1135$ different taxonomies, however after removing taxonomies used by less tham $4$ documents ($\tilde 0.01\%$ of the amount of documents with taxonomies), $613$ remain.

Reviewer: ```suggestion
Originally there were $1135$ different taxonomies however after removing taxonomies used by less than $4$ documents ($\tilde 0.01\%$ of the number of documents with taxonomies), $613$ remain.
```
